src/dsymbol/conversion/second.d

```diff
@@ -277,6 +311,65 @@ do
 		suffix.ownType = false;
 		symbol.type = lastSuffix;
 		symbol.ownType = true;
+		
+		// if that's a pointer, then add the symbols since we built a new one
+		if (lastSuffix.isPointer)
+		{
+			// add symbols based on what pointer is pointing to
+			bool isPtr = false;
+			bool ptrType = false;
+			bool ptrArray = false;
+			bool ptrAA = false;
+
+			auto index = callTip.length;
+			while(index--)
```

Comment:
wouldn't it be better to iterate over the breadcrumbs (maybe make a copy) instead of iterating over the characters for stability?